scst-store/cluster-specific-scanner-configurations.hbs.md

```diff
@@ -0,0 +1,19 @@
+## Connecting vulnerability scanning to Supply Chain Security Tools - Store
+ - Single cluster - Scanners will use `app-tls-cert` to communicate with SCST - Store. Please refer [Full profile Setup](../install.hbs.md#install-profile)
+ - Multicluster … Sacnners will use `ingress-cert` of SCST - Store in view cluster. Please refer [MultiCluster Setup](multicluster-setup.hbs.md)
+
+
+## Grype and other supported scanner configuration
+The configuration to connect the Grype scanner or another supported scanner to SCST - Store lies in the scanner configuration. 
```

Comment:
i think this should go under the header above ("Connecting vulnerability scanning...")... then we can delete this header ("Grype and other...")